templates/web/smidsy/maps/osm-toner-lite.html

```diff
@@ -0,0 +1,15 @@
+[% map_js = BLOCK %]
+<script type="text/javascript" src="[% version('/js/OpenLayers.fixmystreet.js') %]"></script>
+<script type="text/javascript" src="https://stamen-maps.a.ssl.fastly.net/js/tile.stamen.js?v1.3.0"></script>
+<script type="text/javascript" src="[% version('/js/map-OpenLayers.js') %]"></script>
+<script type="text/javascript" src="[% version('/js/map-toner-lite.js') %]"></script>
+<script type="text/javascript" src="[% version('/js/jquery.ba-hashchange.min.js') %]"></script>
+<!--[if lte IE 6]>
+  <link rel="stylesheet" href="/js/OpenLayers-2.13.1/theme/default/ie6-style.css" type="text/css">
+<![endif]-->
+[% END %]
+
+[% map_html = BLOCK %]
+[% INCLUDE maps/openlayers.html %]
+<script type="text/javascript">fixmystreet.show_stats19 = '[% show_stats19 %]';</script>
```

Comment:
Thanks! I've fixed the XSS in 405b0429827519c4831d3b6acc63063de8d0b5cf and added the fms var declaration as a precaution. 

I'm not sure about changing the template though  - this one is quite small already, and the js needs to be here (after the other templated map js is loaded in map/openlayer.html), it seems a fairly obvious place to put it.